_config.yml

```diff
@@ -1,12 +1,12 @@
 # Site settings
-title: 'Temur Fatkulin'
+title: 'Timur Fatkulin'
```

Comment:
@marshall-lee Are you sure? I thought that it was right. 
@istickz what do you say?

Reply:
@thesunwave I don't know for sure, it's a matter for discussion — that's why I opened this PR. I didn't hear of the name "Temur" before so I checked Wikipedia page for "Temur" and it redirected me:

![image](https://user-images.githubusercontent.com/3194530/35814682-b35f946c-0aa7-11e8-8481-5c7d08b83db6.png)

Reply:
nice pull request! Timur sounds better than temur! Just get it!

Reply:
actually, Temur has been waiting for this all his life